syntax = "proto3";

package fixbug;

option cc_generic_services = true;

message ResultCode
{
    int32 errcode = 1;
    bytes errmsg = 2;
}

message GetFriendsListRq
{
    uint32 userid = 1;
}

message GetFriendsListRs
{
    ResultCode result = 1;
    repeated bytes friends = 2;
}



service FriendServiceRpc
{
    rpc GetFriendsList(GetFriendsListRq) returns(GetFriendsListRs);
}